server/controllers/label.js

```diff
@@ -8,4 +8,16 @@ export default class labelController {
     ]);
     return { message: "Label created" };
   };
+
+  static createTaskLabelRelation = async (req) => {
+    for (const label of req.body.labelId) {
+      await pool.query(
+        `INSERT INTO tasks_labels_relation (task_id, label_id) VALUES ($1, $2);`,
+        [req.body.taskId, label],
+      );
+    }
+    return {
+      message: `Task with ID: ${req.body.taskId} is now labeled by label with ID: ${req.body.labelId}`,
```

Comment:
This won't really work for many labels, change it to something general or loop over `req.body.labelId` to get all the label IDs.

Reply:
? Is this comment actual?